app/services/invites/create_service.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Invites
+  class CreateService < BaseService
+    def call(**args)
+      existing_invite = Invite.find_by(
```

Comment:
this should be done in an `Invites::ValidateService` like we do on `Wallet` or `WalletTransaction`

Reply:
Just did in last commit ! Created this ValidateService and moved out all "custom" validations into it